.github/workflows/swagger.yml

```diff
@@ -7,32 +7,29 @@ on:
   pull_request:
     branches:
       - master
+  workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: Set up Cache
-        uses: actions/cache@v1
-        with:
-          path: ~/.gradle/caches
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
-          restore-keys: ${{ runner.os }}-gradle-
+      - uses: actions/checkout@v3
       - name: Set up JDK 11
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
           java-version: 11
+          distribution: 'temurin'
+          cache: 'gradle'
       - name: Run Swagger Publisher
         run: ./gradlew integration --tests uk.gov.hmcts.reform.wataskmanagementapi.config.OpenAPIPublisherTest
       - name: Commit to repository
         run: |
           mkdir swagger-staging
             cd swagger-staging
           git init
-          git config user.email "jenkins-reform-iac@hmcts.net"
-          git config user.name "WA Task Management API"
-          git remote add upstream https://${{ secrets.SWAGGER_PUBLISHER_API_TOKEN }}@github.com/hmcts/reform-api-docs.git
+          git config user.email "github-platform-operations@HMCTS.NET"
+          git config user.name "HMCTS Platform Operations"
+          git remote add upstream https://apitoken:${{ secrets.SWAGGER_PUBLISHER_API_TOKEN }}@github.com/hmcts/cnp-api-docs.git
```

Comment:
the critical change was using a username here, it didn't seem to want to work with token only.